ironfish/src/webApi.ts

```diff
@@ -93,23 +89,15 @@ export class WebApi {
     return response.data
   }
 
-  async getNextFaucetTransactions(count: number): Promise<FaucetTransaction[] | null> {
+  async getNextFaucetTransactions(count: number): Promise<FaucetTransaction[]> {
     this.requireToken()
 
-    try {
-      const response = await axios.get<{ data: FaucetTransaction[] }>(
-        `${this.host}/faucet_transactions/next?count=${count}`,
-        this.options(),
-      )
-
-      return response.data.data
-    } catch (e) {
-      if (IsAxiosError(e) && e.response?.status === 404) {
-        return null
-      }
+    const response = await axios.get<{ data: FaucetTransaction[] }>(
+      `${this.host}/faucet_transactions/next?count=${count}`,
+      this.options(),
+    )
```

Comment:
Is removing this try/catch going to have ill effects if this `get` throws an Error? I suppose the Docker service should restart it even if it crashes. Just curious if there was a bug leading to this change

Reply:
The linter said it was an unnecessary try-catch wrapper after removing the check for a 404 status code. I'm assuming it doesn't, but I'll see if I can get a better answer.